src/Uno.UI/DataBinding/BindingExpression.cs

```diff
@@ -45,6 +47,7 @@ public partial class BindingExpression : IDisposable, IValueChangedListener
 		public Binding ParentBinding { get; }
 
 		internal DependencyPropertyDetails TargetPropertyDetails { get; }
+		internal TemplatedParentScope TemplatedParentScopeDebug => _templatedParentScope;
```

Comment:
we can maybe just keep the instance of TemplatedParent instead, since it will be immediately available.